### PR TITLE
feat: import revolut trades

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: src/parsers/trades/revolut/index.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    'Base currency': string;
+    State: string;
+    Balance: string;
+}
+
+export default async function processData(importDetails: IImport): Promise<ITrade[]> {
+    const data: IRevolut[] = await getCSVData(importDetails.data) as IRevolut[];
+    const internalFormat: ITrade[] = [];
+    for (const trade of data) {
+        if (trade.Type != 'EXCHANGE') {
```
Comment (starsoccer) [inline src/parsers/trades/revolut/index.ts:25]: Small tweak, here from `!=` to `!==`

File: src/parsers/trades/revolut/index.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            console.log(`Not an exchange ${trade.Type}`);
+            continue;
+        }
+        const tradeToAdd: IPartialTrade = {};
+        tradeToAdd.date = createDateAsUTC(new Date(trade['Completed Date'])).getTime();
+        tradeToAdd.exchange = EXCHANGES.Revolut;
+        const bought = trade.Description.substr(13);
```
Comment (starsoccer) [inline src/parsers/trades/revolut/index.ts:32]: Is there a better way we could pull out which currency was bought/sold? Do you have an example description I could look at?

Comment (Zahrun) [reply src/parsers/trades/revolut/index.ts:32]: I will send one to you

Comment (Zahrun) [reply src/parsers/trades/revolut/index.ts:32]: [account-statement_2020-10-01_2022-11-30_en-gb_92e922.csv](https://github.com/starsoccer/cryptotithe/files/10130466/account-statement_2020-10-01_2022-11-30_en-gb_92e922.csv)

Comment (starsoccer) [reply src/parsers/trades/revolut/index.ts:32]: Could we do something like this instead?
```
const bought = trade.Description.substr(trade.Description.lastIndexOf(' '));
```

This should get whatever the last few characters are and wont be as reliant on a magic number

Comment (Zahrun) [reply src/parsers/trades/revolut/index.ts:32]: Sure, let’s do that

Comment (Zahrun) [reply src/parsers/trades/revolut/index.ts]: Please check my force-push. I just added +1 otherwise the space would be taken as well.

**Pull request merged**